### PR TITLE
introduced bytes.as-hash object

### PR DESCRIPTION
**Pull request by Graur**
#740

### CODE REVIEW

**Comment by Graur** [comment]
@yegor256 please, have a look

File: eo-runtime/src/test/eo/org/eolang/bytes-tests.eo [hunk 1]
```diff
@@ -342,3 +342,9 @@
       $.equal-to
         "hello world"
 
+[] > bytes-hashes-equals
```
Comment (yegor256) [inline eo-runtime/src/test/eo/org/eolang/bytes-tests.eo:345]: @Graur good test, but a weak one. I suggest adding one more test, which would compare hash with a value we expect.

**Comment by yegor256** [comment]
@Graur see above

File: eo-runtime/src/test/eo/org/eolang/bytes-tests.eo (hunk 1 above)
Comment (Graur) [reply eo-runtime/src/test/eo/org/eolang/bytes-tests.eo:345]: @yegor256 Agree. I will add a new test, thanks!

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@Graur thanks!

**Pull request merged**